packages/strapi-plugin-upload/admin/src/components/Card/Border.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { themePropTypes } from 'strapi-helper-plugin';
+
+const Border = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: none;
+  ${({ shown }) => shown && 'display: block;'}
```

Comment:
` ${({ shown }) => shown && 'display: block'};`

Reply:
The logical `&&` operator doesn't show the display attribute everytime so I cannot set the `;` outside because it will be displayed each time instead of only when the `display` attribute exists.